python/triton/language/standard.py

```diff
@@ -442,7 +442,6 @@ def interleave(a, b):
     """
     c = core.join(a, b)
 
-    assert isinstance(c.shape, list)
```

Comment:
why are we removing this assert?

Reply:
The `join` method always returns a tensor with a shape of `list` type